CommonTools/RecoAlgos/plugins/PtMinPixelMatchGsfElectronCountFilter.cc

```diff
@@ -1,20 +0,0 @@
-/* \class PtMinPixelMatchGsfElectronCountFilter
- *
- * Filters events if at least N electrons above 
- * a pt cut are present
- *
- * \author: Luca Lista, INFN
- *
- */
-#include "FWCore/Framework/interface/MakerMacros.h"
-#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
-#include "DataFormats/EgammaCandidates/interface/GsfElectronFwd.h"
-#include "CommonTools/UtilAlgos/interface/ObjectCountFilter.h"
-#include "CommonTools/UtilAlgos/interface/PtMinSelector.h"
-
-typedef ObjectCountFilter<
-          reco::GsfElectronCollection, 
-          PtMinSelector
-        >::type PtMinPixelMatchGsfElectronCountFilter;
```

Comment:
In principle with the current implementation of the default label this module could co-exist with `PtMinGsfElectronCountFilter.cc`, but as noted in the commit message the definition is exactly the same
https://github.com/cms-sw/cmssw/blob/e775c2b6b822417a83d14d30a545fc5628003c15/CommonTools/RecoAlgos/plugins/PtMinGsfElectronCountFilter.cc#L15-L18
and there are no uses within CMSSW. My preference would be to keep the deletion commit.